kivy/core/window/_window_sdl2.pyx

```diff
@@ -413,6 +413,7 @@ cdef class _WindowSDL2Storage:
                 windows_info = WindowInfoWindows()
                 windows_info.window = wm_info.info.win.window
                 windows_info.hdc = wm_info.info.win.hdc
+                return windows_info
```

Comment:
Hi @FI-Mihej , it look like the tests written by the previous contributor was not executed, but with your fixes, now they are :)
And they break. Would you mind give a shot and fixes them too ?

I think just changing https://github.com/kivy/kivy/blame/c28c47b39ae57c97c70cc0398d74774d73a6894b/kivy/tests/test_window_info.py#L28 -> using lowercase attribute should work, as the definition of theses variables are lowercase. https://github.com/kivy/kivy/blob/c28c47b39ae57c97c70cc0398d74774d73a6894b/kivy/core/window/window_info.pxd#L18

Thanks,